pdfannots/cli.py

```diff
@@ -11,8 +11,15 @@
 from .printer.json import JsonPrinter
 
 
-MD_FORMAT_ARGS = ['print_filename', 'remove_hyphens', 'wrap_column', 'condense', 'sections']
-"""Named of arguments passed to the markdown printers."""
+MD_FORMAT_ARGS = [
+    'condense',
+    'group',
```

Comment:
This line is basically harmless but incorrect -- "group" was not in the original list because it is consumed by the code below in this file that decides which class to construct

Reply:
I removed it. Don't know why I put it there...